WcaOnRails/lib/city_comma_region_validator.rb

```diff
@@ -4,6 +4,8 @@ class CityCommaRegionValidator < CountryCityValidator
   attr_reader :valid_regions
 
   def initialize(type_of_region:, valid_regions:)
+    super
```

Comment:
I was bugged by the CI error and checked this out locally, turns out you need to do:
```suggestion
    super()
```
But I don't know why there is an argument passed to super otherwise...

Reply:
Thanks for catching the CI error so quickly!